basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
+| Support for MySQL 8.0 clients                                                        | Y     | Y   | N   | N   | N   |
+| [MySQL compatible `GRANT` system](/privilege-management.md)                          | Y     | Y   | Y   | Y   | Y   |
+| Auditing                                                                             | Y     | Y   | Y   | Y   | Y   |
+| **Data import and export**                                                           | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [Fast Importer (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md)         | Y     | Y   | Y   | Y   | Y   |
+| [mydumper logical dumper](/mydumper-overview.md)                                     | Deprecated | Deprecated | Y   | Y   | Y   |
+| [dumpling logical dumper](/dumpling-overview.md)                                     | Y     | Y   | N   | N   | N   |
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)              | Y     | N   | N   | N   | N   |
+| [Database migration toolkit (DM)](/migration-overview.md)                            | Y     | Y   | Y   | Y   | Y   |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                  | Deprecated | Deprecated | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                | Y     | Y   | N   | N   | N   |
+| **Management, observability and tools**                                              | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [TiDB Dashboard](/dashboard/dashboard-intro.md)                                      | Y     | Y   | N   | N   | N   |
+| [SQL diagnostics](/information-schema/information-schema-sql-diagnostics.md)         | Experimental | Experimental | N   | N   | N   |
+| [Information schema](/information-schema/information-schema.md)                      | Y     | Y   | Y   | Y   | Y   |
+| [Metrics schema](/metrics-schema.md)                                                 | Y     | Y   | N   | N   | N   |
+| [Statements summary tables](/statement-summary-tables.md)                            | Y     | Y   | N   | N   | N   |
+| [Slow query log](/identify-slow-queries.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| [TiUP deployment](/tiup/tiup-overview.md)                                            | Y     | Y   | N   | N   | N   |
+| [Ansible deployment](/online-deployment-using-ansible.md)                            | Deprecated | Deprecated | Y   | Y   | Y   |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/)                  | Y     | Y   | Y   | Y   | Y   |
+| [Built-in physical backup](/br/backup-and-restore-use-cases.md)                      | Y     | Y   | N   | N   | N   |
+| Point in time recovery (PITR)                                                        | Planned | N | N   | N   | N   |
```

Comment:
IIUC, PITR is going to be GA in TiDB 5.0. @IANTHEREAL please confirm.

Reply:
As confirmed with @IANTHEREAL, currently it cannot be confirmed and might change.